.github/workflows/infrastructure.yml

```diff
@@ -0,0 +1,171 @@
+        'Team=development' >> $GITHUB_ENV
+
+    - name: Login 🔐
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: Azure defaults ✨
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          # Basic
+          az configure --defaults location=${{ vars.REGION }}
+          az configure --defaults group=rg-${{ env.product }}-${{ env.target }}-001
+
+          # AZ CLI extensions upgrade
+          az extension add --name containerapp --upgrade
+
+    - name: Resource group 🏗️
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az group create \
+          --name rg-${{ env.product }}-${{ env.target }}-001 \
+          --tags ${{ env.tags }}
+
+    
+    - name: Container registry 📦️
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az acr create \
+          --name cr${{ env.product }}${{ env.target }}001 \
+          --sku Standard \
+          --admin-enabled true \
+          --tags ${{ env.tags }}
+
+    - name: Virtual network 🧵
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az network vnet create \
+          --name vnet-${{ env.product }}-001 \
+          --address-prefix ${{ secrets.VNET_ADDRESS_PREFIX }} \
+          --subnet-name snet-${{ env.product }}-001 \
+          --subnet-prefixes ${{ secrets.VNET_SUBNET_PREFIX }} \
+          --tags ${{ env.tags }}
+          
+    # - name: VNET peer 🔀
+    #   uses: Azure/cli@1.0.4
+    #   with:
+    #     inlineScript: |
+    #       az network vnet peering create \
+    #       --name vnet-peer-${{ env.product }}-001 \
+    #       --vnet-name vnet-${{ env.product }}-001 \
+    #       --remote-vnet $(az network vnet show --subscription ${{ secrets.REMOTE_VNET_SUBSCRIPTION }} --resource-group ${{ secrets.VNET_REMOTE_RESOURCE_GROUP }} --name ${{ secrets.REMOTE_VNET }} --query 'id' -o tsv) \
+    #       --allow-forwarded-traffic 1 \
+    #       --allow-vnet-access 1
+
+    - name: Container app environment 🗃️
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az containerapp env create \
+          --name cae-${{ env.product }}-${{ env.target }}-001 \
+          --infrastructure-subnet-resource-id $(az network vnet subnet show --name snet-${{ env.product }}-001 --vnet-name vnet-${{ env.product }}-001 --query 'id' -o tsv) \
+          --tags ${{ env.tags }}
+
+    
+    - name: Container app - MDM 📄
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az containerapp create \
+          --name ca-${{ env.product }}-mdm-${{ env.target }}-001 \
+          --environment cae-${{ env.product }}-${{ env.target }}-001 \
+          --registry-server cr${{ env.product }}${{ env.target }}001.azurecr.io \
+          --registry-username $(az acr credential show -n cr${{ env.product }}${{ env.target }}001 --query username | tr -d '"') \
+          --cpu 1.0 \
+          --memory 2.0Gi \
```

Comment:
I see in a previous [comment](https://github.com/UK-Export-Finance/mdm-api/pull/58/files/6afea90233a247d5798e9940f26b87ea8f807dee#r1141096557) we increased the specs for testing at the moment - do we need a ticket to reduce the specs later or are we fine to keep this indefinitely?